### DISCUSSION TITLE
Phoenix code

### DISCUSSION THREAD

**Discussion by pizzipie** [phcode-dev/phoenix#1615] [Q&A] [answered]
How do you delete a project?

**Comment by abose** [comment, marked as answer]
Projects are just folders in your system . So to delete projects, you can just delete the folder from your system. If you mean how do i remove projects from showing up in history then, there are two options:

1. Click on the project dropdown, hover over any project and click on the close icon for the project you want to remove from history.
![image](https://github.com/phcode-dev/phoenix/assets/5336369/916553ea-2214-4d12-86ae-d59cfe4f52f6) 
2. Select `file-> new Project` and click on the close icon for the project you want to remove from history.
![image](https://github.com/phcode-dev/phoenix/assets/5336369/c40993ee-a56c-4404-91a3-10dbf614e343)

**Comment by charlypa** [comment]
# Method1
![Phoenix_Code_Z21zpPy1de](https://github.com/phcode-dev/phoenix/assets/1525027/8f961949-2a23-4b56-a669-d0babf3d218e)

# Method2
![Phoenix_Code_cE6M3cFii9](https://github.com/phcode-dev/phoenix/assets/1525027/5858fd28-bfa1-441d-a575-a8a5cf5b97a8)
